### ISSUE TITLE
Return ref of tween target

### ISSUE THREAD

**Issue by fitzmode** [bitworking/react-gsap#19]
Hey there. Seems Tween does not return the `ref` of it's children.

Here's a demo https://codesandbox.io/s/wizardly-mclaren-7oswj?file=/src/App.js

**Comment by bitworking** [comment]
Hmm, the Tween needs the ref by his own and overwrites it in this case. I could try to first get the ref and set it with the new ref var. But why exactly you need it?

**Comment by fitzmode** [comment]
I need the `ref` to call methods on the node. Used `div` as a a contrived reproducible example.

**Comment by bitworking** [comment]
Ok let me try later. You also could get the refs with a teenref.getGSAP().targets() call.

Look here: https://bitworking.github.io/react-gsap/src-instructions-low-level-access

**Comment by bitworking** [comment]
I tried it but cannot set the ref, because it's readonly. I saw your lib (https://github.com/fitzmode/use-dynamic-refs). You perhaps can help me with this?

I do the following, which probably could be optimized, too:

https://github.com/bitworking/react-gsap/blob/master/packages/react-gsap/src/Tween.tsx#L204

```javascript

React.Children.map(children, child => {
  return React.cloneElement(child, {
    ref: target => {
      this.addTarget(target);
      // this doesn't work and I don't know if
      // setting the target to child.ref is the way to go
      child.ref = target;
    },
  });
})

```

**Comment by fitzmode** [comment]
Let me work on it and will come back with a PR

**Comment by bitworking** [comment]
I found a solution: https://github.com/facebook/react/issues/8873#issuecomment-489579878

Will be in next version.

**Comment by bitworking** [comment]
Fixed with version 2.3.0
